README.md

```diff
@@ -1,4 +1,748 @@
+		}
+		return nil
+	} // func.end
+```
+
+
+### Updating Kafka topic's configuration
+---
+To increase a Kafka topic's partition number or to update a Kafka topic's configuration, issue a
+`PATCH` request to `/admin/topics/TOPICNAME` with the following body:
+(where TOPICNAME is the name of the Kafka topic that you want to update).
+```json
+{
+  "new_total_partition_count": 4,
+  "configs": [
+    {
+      "name": "cleanup.policy",
+      "value": "compact"
+    }
+  ]
+}
+```
+Supported configuration keys are 'cleanup.policy', 'retention.ms', 'retention.bytes', 'segment.bytes', 'segment.ms', 'segment.index.bytes'.
+And partition number can only be increased, not decreased.
+
+Expected status codes
+- 202: Update topic request was accepted.
+- 400: Invalid request JSON/number of partitions is invalid.
+- 404: Topic specified does not exist.
+- 422: Semantically invalid request.
+
+#### Example
+
+```golang
+	func updateTopicDetails(serviceAPI *adminrestv1.AdminrestV1) error {
+		// Construct an instance of the UpdateTopicOptions model
+		updateTopicOptionsModel := new(adminrestv1.UpdateTopicOptions)
+		updateTopicOptionsModel.TopicName = core.StringPtr("test-topic")
+		updateTopicOptionsModel.NewTotalPartitionCount = core.Int64Ptr(int64(6))
+	
+		// Invoke operation with valid options model.
+		response, operationErr := serviceAPI.UpdateTopic(updateTopicOptionsModel)
+		if operationErr != nil {
+			return fmt.Errorf("Error Updating Topic: %s\n", operationErr.Error())
+		}
+	
+		// Check the result.
+		if response.StatusCode != http.StatusAccepted {
+			return fmt.Errorf("Error Updating Topics: status %d\n", response.StatusCode)
+		}
+	
+		fmt.Printf("\tname: %s updated\n", *updateTopicOptionsModel.TopicName)
+	
+		return nil
+	} // func.end
+```
+
+
+### List current mirroring topic selection
+
+Mirroring user controls are only available on the target cluster in a mirroring environment.
+
+To get the current topic selection, issue an GET request to /admin/mirroring/topic-selection
+
+
+Expected status codes
+- 200: Retrieved topic selection successfully in following format:
+```json
+{
+  "includes": [
+    "^prefix1_.*",
+    "^prefix2_.*"
+  ]
+}
+```
+- 403: Unauthorized to use mirroring user controls.
+- 404: Mirroring not enabled. The mirroring user control APIs are only available on the target cluster of a mirrored pair.
+- 503: An error occurred handling the request.
+
+#### Example
+
+```golang
+	func listMirroringTopicSelection(serviceAPI *adminrestv1.AdminrestV1) error {
+		// Construct an instance of the GetMirroringTopicSelectionOptions model
+		getMirroringTopicSelectionOptionsModel := new(adminrestv1.GetMirroringTopicSelectionOptions)
+	
+		// Call GetMirroringTopicSelection.
+		result, response, operationErr := serviceAPI.GetMirroringTopicSelection(getMirroringTopicSelectionOptionsModel)
+		if operationErr != nil {
+			return fmt.Errorf("Error Listing Mirroring Topics: %s\n", operationErr.Error())
+		} // func.end
```

Comment:
It's missing:
```golang
    return nil
}

Reply:
yeh the //func.end was in the wrong place.